statline_bq/utils.py

```diff
@@ -159,13 +161,83 @@ def get_main_table_shape(metadata: dict) -> dict:
         The dataset's main table's shape
     """
     main_table_shape = {
-        "n_records": get_from_meta(metadata, "RecordCount"),
-        "n_columns": get_from_meta(metadata, "ColumnCount"),
-        "n_observations": get_from_meta(metadata, "ObservationCount"),
+        "n_records": metadata.get("RecordCount"),
+        "n_columns": metadata.get("ColumnCount"),
+        "n_observations": metadata.get("ObservationCount"),
     }
     return main_table_shape
 
 
+def get_schema_cbs(metadata_url, odata_version) -> pa.Schema:
```

Comment:
sweeeeet!!

Reply:
:)
Still needs some work, but I expect this to work with most use cases from CBS.